api/src/geolocate/get.handler.ts

```diff
@@ -21,24 +20,48 @@ async function getGeoDataForLatLong(
   params: SqlParametersList,
   geoid: string,
   table: string,
-): Promise<string | undefined> {
+): Promise<BoundedGeoDatum | undefined> {
+  // TODO(breuch): Consider updating this to all geoids when we support
+  // county, states, etc.
   const executeParams: ExecuteStatementRequest = {
     database: process.env.DATABASE_NAME ?? 'postgres',
     resourceArn: process.env.RESOURCE_ARN ?? '',
     schema: SCHEMA,
     secretArn: process.env.CREDENTIALS_SECRET ?? '',
-    sql: `SELECT ${geoid}
-          FROM ${table}
-          WHERE geom && ST_SetSRID(ST_Point(:long, :lat), 4326)
-          LIMIT 1`,
+    sql: `
+      WITH target_row AS (
```

Comment:
Made this a subquery and explicitly sorted so that at whatever we return from the API is stable.